.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
It is clear that you added linters to assist in formatting the code base, however there were no linter checks on the pull request. I believe it won't bother you double check the linter configurations. Linters are useful in the codebase because as the author will only focus on functionality while syntactic errors are taken care of.

Reply:
@MPDADDY 
I have fixed the linters issue and the check is now complete. 
Kindly review my code.